Maven install issue

After I downloaded the project and ran it with "mvn clean install", it gave me the error below (I am using mac). 

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:25 min
[INFO] Finished at: 2017-02-08T12:09:21-05:00
[INFO] Final Memory: 12M/320M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project eskg: Could not resolve dependencies for project org.esipfed:eskg:jar:1.0-SNAPSHOT: Could not find artifact org.apache.any23.plugins:apache-any23-basic-crawler:jar:2.0-SNAPSHOT -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 

Also, please see my commit here. 
https://github.com/Yongyao/eskg/commit/681aadb18cef3eee6c0f190ffc228032d29310ff

--- a/pom.xml
+++ b/pom.xml
@@ -38,6 +38,19 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <repositories>
+    <repository>
+      <id>apache-snapshots</id>
+      <url>https://repository.apache.org/content/repositories/snapshots/</url>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
   <scm>
     <url>https://github.com/ESIPFed/eskg</url>
     <connection>scm:git:git://github.com/ESIPFed/eskg.git</connection>

